src/main/java/hudson/plugins/accurev/CheckForChanges.java

```diff
@@ -295,4 +192,42 @@ public static boolean checkStreamForChanges(AccurevServer server,
         }
         return isTransLatestThanBuild;
     }
+
+    private static boolean changesMatchFilter(TaskListener listener, Collection<String> serverPaths, List<String> filters) {
+        if (filters == null || filters.isEmpty()) {
+            // No filters, so always a match.
+            return true;
+        }
+
+        for (String filterPath : filters) {
+            // Paths are sanitized using java.nio.file.Paths (makes sure that the same path separators are used)
+            final Path fp = Paths.get(filterPath);
+            for (String serverPath : serverPaths) {
+                final Path sp = Paths.get(serverPath);
+                if (sp.toString().contains(fp.toString())) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    private static List<String> getListOfPollingFilters(String filterForPollSCM, String subPath) {
+        final String DELIMITER = ",";
+        List<String> result = new ArrayList<>();
+
+        if (filterForPollSCM != null && !(filterForPollSCM.isEmpty())) {
+            for (String filter : filterForPollSCM.split(DELIMITER)) {
+                result.add(filter.trim());
+            }
+        } else {
+            if (subPath != null && !(subPath.isEmpty())) {
```

Comment:
Current client does not allow me to write an `else` statement without braces. I guess it's influencing my other work tool. I'll apply your suggestion though. ;)